miso-web/src/spring-test/java/uk/ac/bbsrc/tgac/miso/webapp/springtest/AbstractST.java

```diff
@@ -212,7 +212,8 @@ private String pollingResponserHelper(String requestType, List<?> dtos, String c
       case "put":
         mvcResult = getMockMvc()
             .perform(put(controllerBase + "/bulk").contentType(MediaType.APPLICATION_JSON).content(makeJson(dtos)))
-            .andExpect(status().isAccepted())
+            .andDo(print())
+            .andExpect(status().is(isOneOf(202, 204))) // qc status gets 204 from this request so this is needed
```

Comment:
This would mean any of the other endpoints checked through this method could change from 202 to 204 without failing the test.

Reply:
`QcStatusRestController` doesn't do async saving, so this method should not be adapted to handle it